bigchaindb/backend/schema.py

```diff
@@ -122,7 +122,7 @@ def validate_language_key(obj, key):
     """
     backend = bigchaindb.config['database']['backend']
 
-    if backend == 'mongodb':
+    if backend in ('mongodb', 'localmongodb'):
```

Comment:
I think we can just use `'localmongodb'`, @vrde ?

Reply:
Why would you only want to validate the language in tests?

Reply:
@ldmberman The 'mongodb' backend will be gone after refactoring. So, I think we can just use `'localmongdb'` there.

Reply:
I understand now, thanks! Yes, makes sense to only keep `localmongodb` then.